docs/nodes/consensus-node.mdx

```diff
@@ -143,53 +209,78 @@ cd $HOME<br/>
 rm -rf ~/.celestia-app/data<br/>
 mkdir -p ~/.celestia-app/data<br/>
 SNAP_NAME=$(curl -s https://snaps.qubelabs.io/celestia/ | \<br/>
-&nbsp; egrep -o ">{constants.mochaChainId}.*tar" | tr -d ">")<br/>
+    egrep -o ">{constants.mochaChainId}.*tar" | tr -d ">")<br/>
 wget -O - https://snaps.qubelabs.io/celestia/$&#123;SNAP_NAME&#125; | tar xf - \<br/>
-&nbsp; -C ~/.celestia-app/data/<br/>
+    -C ~/.celestia-app/data/<br/>
 </code></pre>
 
 </TabItem>
-<TabItem value="blockspacerace" label="Blockspace Race">
+<TabItem value="arabica" label="Arabica">
 
-Run the following command to quick-sync from a snapshot for `blockspacerace`:
+Run the following command to quick-sync from a snapshot for `arabica-10`:
```

Comment:
[question] is it possible to interpolate the Arabica chain-id here? Maybe something like: `{constants.arabicaChainId}`?

Reply:
yep! using the inline text component:

```mdx
This is a sentence with Arabica's chain-id: <InlineText constant={constants.arabicaChainId} />.
```

Reply:
48ad87f67c300b20a5254866d1eaa27c3838e9a1 great catch too, fixed on the other instance as well